graph/patterns/dictionary.md

```diff
@@ -2,24 +2,24 @@
 
 Microsoft Graph API Design Pattern
 
-*The dictionary type provides the ability to create a set of primitives or objects of the same type where the API consumer can define a name for each value in the set.*
+_The dictionary type provides the ability to create a set key/value pairs where the set of keys is dynamically specified by the API consumer._
 
 ## Problem
 
-The API design requires a resource to include an unknown quantity of data elements of the same type that must be named by using values provided by the API consumer.
+The API design requires a resource to include an unknown quantity of data values whose keys are defined by the API consumer.
 
 ## Solution
 
-API designers use a JSON object to represent a dictionary in an `application/json` response payload. When describing the model in CSDL, a new complex type can be created that derives from `Org.OData.Core.V1.Dictionary` and then uses the `Org.OData.Validation.V1.OpenPropertyTypeConstraint` to constrain the type that can be used for the values in the dictionary.
+API designers use a JSON object to represent a dictionary in an `application/json` response payload. When describing the model in CSDL, a new complex type can be created that derives from `graph.Dictionary` and optionally uses the `Org.OData.Validation.V1.OpenPropertyTypeConstraint` to constrain the type that can be used for the values in the dictionary as appropriate.
```

Comment:
graph.Dictionary doesn't exist in a workload's CSDL.  So what are they supposed to use here?

I think we still have them create the Core Dictionary type, and suppress the conflict.  See https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads?path=%2FWorkloads%2FMicrosoft.DataClassificationService%2Foverride%2Fschema-Prod-beta.csdl&_a=contents&version=GBmaster for example.

Reply:
> So what are they supposed to use here?

Just definite it, like any other shared complex type.

> I think we still have them create the Core Dictionary type

That would need an AGS change. Core.Dictionary is not supported today.